src/common/i18n/cim-i18n.tsx

```diff
@@ -0,0 +1,5 @@
+import { useTranslation as useReactI18NextTranslation } from 'react-i18next';
+
+export function useTranslation() {
+  return useReactI18NextTranslation(`${process.env.TRANSLATION_NAMESPACE}`);
```

Comment:
Im not sure what happens when we run this in OCM and `TRANSLATION_NAMESPACE` is undefined ..does it work fine ?

Reply:
I don't know. Here we can get more info abou internationalizaiton: https://github.com/openshift/console/blob/master/INTERNATIONALIZATION.md

Reply:
Can you try it in OCM ? It will be undefined there, right ?

Reply:
In OCM don't work. I need to try

Reply:
I think we can use:
export function useTranslation() {
  try {
    return useReactI18NextTranslation(`${process.env.TRANSLATION_NAMESPACE}`);
  } catch (err) {
    return useReactI18NextTranslation('assisted_installer');
  }
}